tests/commonlib/io_utils.py

```diff
@@ -104,7 +104,7 @@ def exec_command(container_name: str, command: str, param_value: str, resource:
                 f"File {resource} does not exist or mount missing.")
 
         if command == 'chmod':
-            os.chmod(path=resource, mode=int(param_value))
+            os.chmod(path=resource, mode=int(param_value, base=8))
```

Comment:
int() is removing the leading zero and as `os.chmod` is expecting an octal value it results with wrong permissions.
Example:
```
int('0664') = 644
os.chmod(filepath, 644)
```
Causing the file permissions to be:
`-w--wx--- (230)`